app/models/rental.rb

```diff
@@ -0,0 +1,34 @@
+class Rental < ApplicationRecord
+  belongs_to :movie
+  belongs_to :customer
+  
+  def decrease_available
```

Comment:
I like that you moved some business logic here. 

However it's not a good idea to keep the available movies in a field, but rather calculate from the total inventory and the number of outstanding rentals.  Otherwise it's possible for things to get out of sync if other applications use the same database.